README.md

```diff
@@ -64,16 +64,94 @@ Histogram can be used to visualize parameter distribution and trends for any ten
 To give the VisualDL a quick test, please use the following commands.
 
 ```
-# Install the VisualDL. Preferably under a virtual environment.
+# Install the VisualDL. Preferably under a virtual environment or anaconda.
 pip install --upgrade visualdl
 
 # run a demo, vdl_create_scratch_log will create logs for testing.
 vdl_create_scratch_log
-visualDL --logdir=scratch_log --port=8080
+visualdl --logdir=scratch_log --port=8080
```

Comment:
I am not sure if you need to change the upper case to lower case, It seems some system will default to lower case and some don't